scraper/text_cleaning.py

```diff
@@ -0,0 +1,25 @@
+def clean_chapter(chapter: dict) -> dict:
+    if "title" in chapter:
+        chapter["title"] = clean_title(chapter["title"])
+    
+    if "content" in chapter:
+        chapter["content"] = clean_paragraph(chapter["content"])
+    
+    return chapter
+
+
+def clean_title(title: str) -> str:
+    return title.replace("\n", "").strip()
+
+
+def clean_paragraph(paragraph: str) -> str:
```

Comment:
not a paragraph. Paragraphs (or scenes) are post splitting.
Call this chapterText or chapterContent